tests/test_runner/test_parallel.py

```diff
@@ -10,6 +10,13 @@
     tblib = None
 
 
+class ErrorThatFailsUnpickling(Exception):
+    def __init__(self, arg1, arg2):
+        self.arg1 = arg1
+        self.arg2 = arg2
+        super(ErrorThatFailsUnpickling, self).__init__("Message with %s and %s")
```

Comment:
Usually we're using single quotes for strings unless the string contains single quotes.